### PR TITLE
Master

### PR DESCRIPTION
**Pull request by thokra-nav**

### CODE REVIEW

**Review by Starefossen — APPROVED** [review]

**Comment by Starefossen** [comment]
Vi får huske å slette master branch etterpå.

File: docs/nais-application/debugging.md [hunk 1]
```diff
@@ -3,8 +3,39 @@ If you have problems getting your pods running you should check out the official
...
+
+Set `JAVA_OPTS` to `-XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/tmp`
+
+The `/tmp` volume is maintained through restarts, so if your app is restarting because of OOM, the heap dumps can be retrieved from there.
+
+## Manually on-demand
+
+You can use `jmap` to create a heap dump of a running Java process.
+
+Find a pod and exec `jmap` in it (considering PID 1 is the Java process):
```
Comment (chinatsu) [inline docs/nais-application/debugging.md:24]:
```suggestion
Find a pod and exec `jmap` in it (assuming PID 1 is the Java process):
```

File: docs/nais-application/debugging.md [hunk 2]
```diff
@@ -3,8 +3,39 @@ If you have problems getting your pods running you should check out the official
...
 
-PS: Applications in the context above is not the NAIS applications. Debugging a NAIS applications resource is done
-with `kubectl describe application $app_name`.
+
+PS: Applications in the context above is not the NAIS applications.
+Debugging a NAIS applications resource is done with `kubectl describe application $app_name`.
+
+# Debugging Memory Leaks
+
+If you experience memory leaks you can get heap dumps either automatically on OOM or on-demand.
```
Comment (chinatsu) [inline docs/nais-application/debugging.md:12]:
```suggestion
If you experience memory leaks in JVM applications you can get heap dumps either automatically on OOM or on-demand.

```

**Review by chinatsu — COMMENTED** [review]
nitpicks :sweat_smile:

**Comment by chinatsu** [comment]
oops, i was too late

**Pull request merged**